ibm/config.go

```diff
@@ -2025,6 +2019,28 @@ func (c *Config) ClientSession() (interface{}, error) {
 	}
 	session.iamPolicyManagementAPI = iamPolicyManagementClient
 
+	// ag
+	iamAccessGroupsURL := iamaccessgroups.DefaultServiceURL
+	if c.Visibility == "private" || c.Visibility == "public-and-private" {
+		if c.Region == "us-south" || c.Region == "us-east" {
+			iamAccessGroupsURL = contructEndpoint(fmt.Sprintf("private.%s.iam", c.Region), cloudEndpoint)
+		} else {
+			iamAccessGroupsURL = contructEndpoint("private.iam", cloudEndpoint)
+		}
+	}
+	iamAccessGroupsOptions := &iamaccessgroups.IamAccessGroupsV2Options{
+		Authenticator: authenticator,
+		URL:           envFallBack([]string{"IBMCLOUD_IAM_API_ENDPOINT"}, iamAccessGroupsURL),
+	}
+	iamAccessGroupsClient, err := iamaccessgroups.NewIamAccessGroupsV2(iamAccessGroupsOptions)
+	if err != nil {
+		session.vpcErr = fmt.Errorf("Error occured while configuring IAM Policy Management service: %q", err)
```

Comment:
@narinder-kaur , It should be
session.iamAccessGroupsErr = fmt.Errorf("Error occured while configuring IAM Access Group service: %q", err)